### DISCUSSION TITLE
Urgent | Sharp fails with standalone docker image

### DISCUSSION THREAD

**Discussion by smo043** [vercel/next.js#36024] [Help]
next.js version: 12.1.4
Docker node image - 16:alpine

Hello,

I am using standalone feature to reduce bundle size and app runs inside a docker container and deployed to azure app service.

Sharp is listed in app dependencies and it exists in the standalone node_modules folder, but as soon the server starts it gives an error 'sharp is required to be installed in standalone mode for image optimization to function correctly'.

I tried all the solutions adding env variable in docker image etc.. but none of the solutions helped.

Can someone help here.

**Comment by WoraponU** [comment]
I Got a same error, what should i config NEXT_SHARP_PATH?

For now I set next.config.js to output: 'standalone' and used custom server.js

**Comment by tomekki92** [comment]
Experiencing this issue as well. Any solutions?

**Comment by JunlinPan-Crypto** [comment]
Same error. Setting `ENV NEXT_SHARP_PATH /app/node_modules/sharp` in dockerfile didn't work either. 
Ref: https://nextjs.org/docs/messages/sharp-missing-in-production

Same issue discussion: https://github.com/vercel/next.js/discussions/35296

**Reply by kunalkeshan** [reply]
Did you find a solution for this?

**Reply by smo043** [reply]
@kunalkeshan - It is no longer an issue since Next 13.
